apps/order/schema.py

```diff
@@ -96,6 +98,79 @@ class Meta:
         filterset_class = OrderFilterSet
 
 
+class OrderTypeForStatistic(DjangoObjectType):
+    total_quantity = graphene.Int()
+
+    class Meta:
+        model = Order
+        fields = ('id', 'order_code', 'total_price')
+
+    def resolve_total_quantity(root, info, **kwargs):
+        return root.book_order.aggregate(Sum('quantity'))['quantity__sum']
```

Comment:
N+1.

Reply:
@thenav56  Actually we don't need this, now I used annotated field, I removed this part